data/schemas/io.elementary.files.gschema.xml

```diff
@@ -39,6 +39,11 @@
       <summary>Thumbnail local files</summary>
       <description>Show thumbnail files on the local filesystem</description>
     </key>
+    <key type="b" name="show-file-preview">
+      <default>false</default>
+      <summary>Show file preview</summary>
+      <description>Show file preview on selection in Columns</description>
```

Comment:
Probably do not need to mention Columns view if the intention is to implement in all views but it can be changed later if preferred